2696. Minimum String Length After Removing Substrings/solution.py

```diff
@@ -0,0 +1,29 @@
+class Solution:
+    # Given a string of uppercase letters, this function removes AB and CD strings as much
+    # as possible and then returns the length of the string. note that removing an AB or
+    # CD string might produce a new one.
+    def minLength(self, s: str) -> int:
+        # what is the index of the first character we are considering?
+
+        # iterate until the index is 2 characters from the length of the string
+
+            # if the pair of characters is AB or CD...
+
+                # remove the characters...
+
+                # and if this is not the first index...
+
+                    # decrement the index so that next iteration, we consider whether
```

Comment:
Ooo this is clever. I did not think of this...

Reply:
For some reason I can't resolve this comment.

Reply:
I meant "this conversation."